api/WorkerNavigator.json

```
@@ -753,6 +897,54 @@
           }
         }
       },
+      "usb": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/webusb/#dom-workernavigator-usb",
+          "support": {
+            "chrome": {
+              "version_added": "69"
```

https://storage.googleapis.com/chromium-find-releases-static/627.html#6279f700aae1f5edbc51e8dacfabab7efce95d2f says 70 and https://bugs.chromium.org/p/chromium/issues/detail?id=837406#c15 also suggests it missed the 69 branch.

It looks like what happened is that *just* `navigator.usb` accidentally shipped in Chrome 69 in https://bugs.chromium.org/p/chromium/issues/detail?id=837406#c15

Let's mark this as supported for "70" anyway and pretend that little mishap didn't happen:

 ```suggestion
              "version_added": "70"
```

Can you mirror that to other browsers?